doc/releases/changelog-dev.md

```diff
@@ -267,13 +267,8 @@
   * :class:`~.PCPhase`
     [(#7591)](https://github.com/PennyLaneAI/pennylane/pull/7591)
 
-<<<<<<< qpe-decomp
-  * :class:`~.QuantumPhaseEstimation`
-    [(#7637)](https://github.com/PennyLaneAI/pennylane/pull/7637)
-=======
   * :class:`~.BasisRotation`
     [(#7074)](https://github.com/PennyLaneAI/pennylane/pull/7074)
->>>>>>> master
```

Comment:
@willjmax I'll let you fix this in your PR [product-formula](https://github.com/PennyLaneAI/pennylane/tree/product-formula)